exporter/importables.go

```diff
@@ -45,39 +46,31 @@ type dbsqlListResponse struct {
 }
 
 // Generic function to list objects related to the DBSQL
-func dbsqlListObjects(ic *importContext, path string) ([]map[string]interface{}, error) {
+func dbsqlListObjects(ic *importContext, path string) (events []map[string]interface{}, err error) {
```

Comment:
@alexott please fix the pagination to return concrete slices, not `[]map[string]interface{}`